bridge_adaptivity/module/urls.py

```diff
@@ -57,5 +57,8 @@
     url(r'^$', RedirectView.as_view(url=reverse_lazy('module:collection-list'))),
 
     # Source outcome service endpoint
-    url(r'^callback_grade/$', callback_sequence_item_grade, name='sequence-item-grade')
+    url(r'^callback_grade/$', callback_sequence_item_grade, name='sequence-item-grade'),
```

Comment:
add `?` in the end of URL to make it work well without trailing slash

` url(r'^callback_grade/?$', callback_sequence_item_grade, name='sequence-item-grade'),`

Reply:
Its internal endpoint which won't be manipulated manually.

Reply:
ok